exs/mundo_1/ruby/003.rb

```diff
@@ -0,0 +1,18 @@
+=begin
+Desafio 003
+
+Problema: Crie um programa que leia dois números e mostre a soma entre eles.
+
+esolução do problema:
```

Comment:
Faltou uma letrinha aqui

```suggestion
Resolução do problema:
```